src/iarc7_motion/iarc_tasks/height_holder.py

```diff
@@ -30,13 +33,17 @@ def __init__(self):
     # that is set as the param _track_roomba_height
     def get_height_hold_response(self, height):
         with self._lock:
-            delta_z = self._TRACK_HEIGHT - height
-            response = self._K_Z * delta_z
+            if self._current_height is None:
+                self._current_height = height
+
+            if not self._USE_CURRENT_HEIGHT:
+                self._delta_z = self._TRACK_HEIGHT - height
```

Comment:
We don't need current height or track height. We only need something called hold_height. When the class is created the height to hold can be passed into the constructor. The param that loads the roomba track height doesn't really belong here IMO.

We could compromise and allow the track height param to be loaded in the constructor. If a height is passed to the constructor then the new variable hold_height can be set to the passed in height or it can be set to the track height parameter.